recipes/pypalettes/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: pypalettes
+  version: "0.1.4"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name | replace("-", "_") }}/${{ name | replace("-", "_") }}-${{ version }}.tar.gz
+  sha256: 3457261c944a4c663035241c7d32001ec29e1e306aaeae1f5790fa5411e1c9d8
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - matplotlib-base >=3.9.2,<4.0
+    - pandas >=2.2.2,<3.0
+    - bs4 >=4.12.3,<5.0
+
+tests:
+  - python:
+      imports:
+      - pypalettes
+      pip_check: true
+      python_version: ${{ python_min }}.*
+  - script:
+      - pytest
+    requirements:
+      run:
+        - pytest
```

Comment:
I'm not 100% sure, but I think we probably want `${{ python_min }}.*` under the `run`, unless it's being picked up from the `python` section. I'm not sure if `scripts` and `python` are independent.

Reply:
Absolutely right, it takes the run requirements.